app/models/git/exercise.rb

```diff
@@ -14,7 +14,6 @@ def self.for_solution(solution)
       )
     end
 
-    # TODO: repo_url can be removed once we're out of a monorepo
```

Comment:
The `repo_url` actually couldn't be removed after all, as it is used to simplify testing.